webrender/src/device/gfx/device.rs

```diff
@@ -2020,6 +2158,38 @@ impl<B: hal::Backend> Device<B> {
         }
     }
 
+    pub fn copy_cache_buffer(&mut self, dst: &Texture, src: &Texture) {
+        let ref src_buffer = self.gpu_cache_buffers[&src.id];
+        let ref dst_buffer = self.gpu_cache_buffers[&dst.id];
```

Comment:
let's assert on both src and dst to be `is_buffer`

Reply:
Fixed here: https://github.com/szeged/webrender/pull/297/commits/76b255b4b7ec31e659242e04dbd757177777abf4#diff-cdf03c96bd34cb98c2d09b0be7658232R2188